app.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const { getCategories } = require("./controllers/categories");
-const { getReviews } = require("./controllers/reviews");
+const { getReviews, getReviewById } = require("./controllers/reviews");
 const app = express();
 
 app.use(express.json());

```

Comment:
You shouldn't have express.json() in the app.js yet - please remove this and only add it in when the code needs it to function